spec/classes/profile/haproxy_spec.rb

```diff
@@ -213,6 +213,17 @@
           end
         end
 
+        context 'with an extra_floating_ip set to 10.0.1.2' do
+          let(:params) do
+            { extra_floating_ips: { 'extra' => '10.0.1.2' } }
+          end
+
+          it do
+            is_expected.to contain_concat_fragment('keepalived preamble')
+              .with_content(%r{virtual_ipaddress {\n\s*10\.0\.1\.2\n\s*}}m)
```

Comment:
This ensures that `nebula::profile::haproxy::extra_floating_ips` makes it into the keepalived file.